.netlifyredirects

```diff
@@ -0,0 +1,6 @@
+/internal   /api
+/tomster    /mascots
+/tomster/*  /mascots/:splat
+/dasboard   /statusboard
+/guides     https://guides.emberjs.com/
+/guides/    https://guides.emberjs.com/
```

Comment:
This needs to have "splatributes" added to the definition. You can see the documentation here: https://www.netlify.com/docs/redirects/#splats